pkg/performanceprofile/controller/performanceprofile/hypershift/hypershift.go

```diff
@@ -235,3 +266,7 @@ func DecodeManifest(b []byte, scheme *runtime.Scheme, obj runtime.Object) (bool,
 	}
 	return false, err
 }
+
+func GetStatusConfigMapName(instanceName string) string {
```

Comment:
minor./nit: `get` or `make`?
didn't we have something similar or the same code elsewhere?

Reply:
We do, I made this one: https://github.com/openshift/cluster-node-tuning-operator/pull/1089#discussion_r1685683711 public